pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -168,7 +175,7 @@ func (r *InferenceServiceReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		reconcilers = append(reconcilers, components.NewExplainer(r.Client, r.Scheme, isvcConfig))
 	}
 	for _, reconciler := range reconcilers {
-		if err := reconciler.Reconcile(isvc); err != nil {
+		if _, err := reconciler.Reconcile(isvc); err != nil {
```

Comment:
Question: is the Result with the `RequeueAfter` propagated out? Seems like we would just ignore the returned Requeue Result here.

Reply:
Good catch! Add code to propagate requeue result to isvc reconcile.